src/components/forms/summit-form.js

```diff
@@ -778,6 +773,96 @@ class SummitForm extends React.Component {
+                            </label>
+                            <textarea
+                                className="form-control"
+                                rows={5}
+                                maxLength={255}
+                                id="REG_LITE_INITIAL_ORDER_COMPLETE_STEP_1ST_PARAGRAPH"
+                                value={regLiteMarketingSettings?.REG_LITE_INITIAL_ORDER_COMPLETE_STEP_1ST_PARAGRAPH?.value}
+                                onChange={this.handleChange}
+                            />
+                            <span className='character-counter'>{`${regLiteMarketingSettings?.REG_LITE_INITIAL_ORDER_COMPLETE_STEP_1ST_PARAGRAPH?.value?.length}/255`}</span>
+                        </div>
+                        <div className="col-md-6">
+                            <label>
+                                {T.translate("edit_summit.reg_lite_initial_order_complete_step_2nd_paragraph")}
+                            </label>
+                            <textarea
+                                className="form-control"
+                                rows={5}
+                                maxLength={255}
+                                id="REG_LITE_INITIAL_ORDER_COMPLETE_STEP_2ND_PARAGRAPH"
+                                value={regLiteMarketingSettings?.REG_LITE_INITIAL_ORDER_COMPLETE_STEP_2ND_PARAGRAPH?.value}
+                                onChange={this.handleChange}
+                            />
+                            <span className='character-counter'>{`${regLiteMarketingSettings?.REG_LITE_INITIAL_ORDER_COMPLETE_STEP_2ND_PARAGRAPH?.value?.length}/255`}</span>
+                        </div>
+                    </div>
+                    <div className="row form-group">
+                        <div className="col-md-6">
+                            <label>
+                                {T.translate("edit_summit.reg_lite_initial_order_complete_btn_label")}
+                            </label>
+                            <Input
+                                className="form-control"
+                                maxLength={50}
+                                id="REG_LITE_INITIAL_ORDER_COMPLETE_BTN_LABEL"
+                                value={regLiteMarketingSettings?.REG_LITE_INITIAL_ORDER_COMPLETE_BTN_LABEL?.value}
+                                onChange={this.handleChange}
+                            />
+                            <span className='character-counter'>{`${regLiteMarketingSettings?.REG_LITE_INITIAL_ORDER_COMPLETE_BTN_LABEL?.value?.length}/50`}</span>
+                        </div>
+                    </div>
+                    <div className="row form-group">
+                        <div className="col-md-6">
+                            <label>
+                                {T.translate("edit_summit.reg_lite_order_complete_step_1st_paragraph")}
+                            </label>
+                            <textarea
```

Comment:
could we create a component here https://github.com/fntechgit/summit-admin/tree/master/src/components/inputs
called TextAreaInputWithCounter
props
className
maxLength
id
value
onChange
and encapsulate the counter logic there ?